### PR TITLE
Update Traefik Chart

### PR DESCRIPTION
**Pull request by ikenna**

### CODE REVIEW

File: apps/base/traefik-hub/repository.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: traefik-hub
 spec:
   interval: 5m
-  url: https://traefik.github.io/charts
+  url: https://helm.traefik.io/traefik
```
Comment (mloiseleur) [inline apps/base/traefik-hub/repository.yaml:8]:
```suggestion
  url: https://traefik.github.io/charts
```
This is the correct address, as documented in [README of the chart](https://github.com/traefik/traefik-helm-chart?tab=readme-ov-file#prerequisites).

**Review by darkweaver87 — COMMENTED** [review]
To generate traffic, with this new version of traefik-hub, [this](https://github.com/traefik-workshops/traefik-hub-gitops/blob/master/apps/base/traffic/traffic.yaml#L49) doesn't work anymore as `ApiGateway` is not part of the CRD anymore. You'll need to create an `IngressRoute` to generate traffic and see something in grafana.

**Comment by ikenna** [comment]
Taking a look. Any changes will be over the weekend.

**Comment by ikenna** [comment]
@darkweaver87 @mloiseleur PR updated. 

Test result screenshot
image

**Review by mloiseleur — APPROVED** [review]

**Review by darkweaver87 — APPROVED** [review]
LGTM

**Pull request merged**